packages/eslint-plugin/src/rules/no-unused-vars-experimental.ts

```diff
@@ -68,7 +68,7 @@ export default util.createRule<Options, MessageIds>({
     },
   ],
   create(context, [userOptions]) {
-    const parserServices = util.getParserServices(context);
+    const parserServices = util.getParserServices(context, true);
```

Comment:
I can't comment on the line above, but you can remove `requiresTypeChecking` from this rule's metadata now.

Reply:
its not that simple, unit test has to be updated first :>